src/language-js/print/typescript.js

```diff
@@ -297,6 +297,9 @@ function printTypescript(path, options, print) {
     case "TSCallSignatureDeclaration":
     case "TSConstructorType": {
       if (n.type !== "TSCallSignatureDeclaration") {
+        if (n.abstract) {
```

Comment:
There are three types in this `case` branch, is it available on the other two types? If not, please add an `if`.